### ISSUE TITLE
Please mark this module as deprecated and redirect to puppetlabs/puppetlabs-docker

### ISSUE THREAD

**Issue by gdubicki** [garethr/garethr-docker#743]
As https://github.com/puppetlabs/puppetlabs-docker is an officially maintained fork of this module it would be nice to edit this module README / description to clearly point everyone to it.

**Comment by marcofl** [comment]
I totally agree with this. Having 2 widely used modules which have same set of parameters but working in different ways is quite "dangerous". We're migrating everything to puppetlabs-docker now
